dayrepo/snippets/views.py

```diff
@@ -71,7 +71,7 @@ def post(self, request):
         print(request.POST)
         req = request.POST
 
-        # Snippets form
+        # Snippets form　スニペットフォーム保存
```

Comment:
```suggestion
        # スニペットフォーム保存
```
英語と日本語で意味を重複させる意味はない

Reply:
あ、日本語のみで許されるんだ、じゃあええか